test/run_tests.pl

```diff
@@ -31,7 +31,7 @@ BEGIN
 my $recipesdir = catdir($srctop, "test", "recipes");
 my $libdir = rel2abs(catdir($srctop, "util", "perl"));
 
-$ENV{OPENSSL_CONF} = rel2abs(catdir($srctop, "apps", "openssl.cnf"));
+$ENV{OPENSSL_CONF} = rel2abs(catdir($srctop, "test", "openssl.cnf"));
```

Comment:
should it be "default.cnf" ?
There should be a comment in openssl.cnf explaining that it is empty on purpose.

Reply:
The empty file has a comment now.
There was already a default.cnf so I renamed it to default-provider.cnf (because it appears to be provider related) so I can have a default.cnf which is empty (minus the comment).